internal/ternary/ternary.go

```diff
@@ -0,0 +1,9 @@
+package ternary
```

Comment:
why is this in the PR? Also, adding a ternary is a bit of a strange thing in golang as it feels like it should be a language level thing

Reply:
Yeah I was using it initially but not anymore. Removed.